.github/dependabot.yml

```diff
@@ -10,3 +10,13 @@ updates:
   assignees:
   - frett
   - levieggertcru
+- package-ecosystem: swift
```

Comment:
The ticket you linked mentioned that the swift support is only for Package.swift and not XCode.

That's similar support that renovate has which doesn't currently cover apps using spm

Reply:
relevant dependabot ticket: https://github.com/dependabot/dependabot-core/issues/7694

Reply:
[Ok, I thought I found something.](https://github.com/dependabot/dependabot-core/pull/7525)

Reply:
@levieggertcru I will leave it up to you whether or not to merge it in.